modules/express/package.json

```diff
@@ -18,7 +22,8 @@
     "@phenyl/http-rules": "~1.0.0",
```

Comment:
I think we should take advantage of Lerna,
how about this?
```
@phenyl/interfaces: "*",
@phenyl/utils: "*",
@phenyl/http-rules: "*",
```

Reply:
version number of lerna monorepo packages are automatically updated using `lerna publish`.
This command also updates the version number on `package.json`, so we can continue using the exact version number.
https://github.com/lerna/lerna/tree/master/commands/publish

Reply:
got it!